apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -190,6 +190,15 @@ const CurriculumCatalog = ({
 
     setNumFilteredTranslatedCurricula(newNumFilteredTranslatedCurricula);
     setFilteredCurricula(newFilteredCurricula);
+
+    if (newNumFilteredTranslatedCurricula === 0) {
```

Comment:
What happens here if a user doesn't need to see only the translated ones? I'm envisioning a scenario where newNumFilteredTranslatedCurricula is 0 but they're still seeing offerings because they don't have that filter

Reply:
Oh good point! I confused myself here for sure, I'll switch it to `newFilteredCurricula.length` rather than `newNumFilteredTranslatedCurricula`